lib/pages/chat/events/message_content.dart

```diff
@@ -317,9 +317,11 @@ class MessageContent extends StatelessWidget {
                     pangeaMessageEvent != null &&
                     !(toolbarController!.highlighted) &&
                     !selected) {
-                  toolbarController!.controller.onSelectMessage(
```

Comment:
You can delete these lines - also, they're already wrapped in a much large pangea comment block so don't worry about adding pangea comments